LiteCore/Support/Error.cc

```diff
@@ -63,8 +63,19 @@ namespace litecore {
         char buf[1024];
         buf[0] = '\x0';
         FormatMessageA(FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS,
-			nullptr, err, MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),
-			buf, sizeof(buf), nullptr);
+            nullptr, err, MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),
+            buf, sizeof(buf), nullptr);
+
+        char* c = buf;
+        while(c) {
```

Comment:
That should be `while (*c)`. The current code will crash if the message doesn't contain a CR; is it untested, or do the tests just get lucky?

Reply:
The tests got lucky!  I put this in because of an awkward newline in the messages.  Nice catch!